tradeit-android-sdk/src/main/java/it/trade/android/sdk/manager/TradeItLinkedBrokerManager.java

```diff
@@ -75,20 +75,48 @@ public void onError(TradeItErrorResult error) {}
         });
     }
 
+    public void syncLinkedBrokers(List<TradeItLinkedLoginParcelable> linkedLoginParcelables) throws TradeItSaveLinkedLoginException, TradeItDeleteLinkedLoginException {
+        List<TradeItLinkedBrokerParcelable> linkedBrokers = this.linkedBrokers;
+
+        // Add missing linkedBrokers
+        for (TradeItLinkedLoginParcelable linkedLoginParcelable: linkedLoginParcelables) {
+            TradeItLinkedBrokerParcelable linkedBrokerParcelable = createNewLinkedBroker(linkedLoginParcelable);
+            if (!linkedBrokers.contains(linkedBrokerParcelable)) {
```

Comment:
We will eventually need to match on both `userId` AND `userToken` because another app instance may have relinked that `userId` to generate a new `userToken`and we would want to sync it...